index.bs

```diff
@@ -1104,6 +1112,29 @@ a minimum of friction, e.g., the user will not have to dig around in their pocke
 example of the latter, when the user is accessing the [=[RP]=] from a given client for the first time, they may be required to
 use a [=roaming authenticator=] which was originally registered with the [=[RP]=] using a different client.
 
+### User Verification Protection enumeration (enum <dfn enum>UserVerificationProtection</dfn>) ### {#user-verification-protection}
+
+<pre class="idl">
+    enum MatcherProtection {
+       "rich-operating-system-software", // need to break Rich-OS kernel in order to by-pass user verification for miusing the key
+       "trusted-execution-environment",  // need to break the Trusted-Execution-Environment 
+                                         //  in order to by-pass user verification for miusing the key
+       "external-chip"};                 // need to break the external chip in order to by-pass user verification for miusing the key
+</pre>
+
+The term <dfn>trusted execution environment</dfn> (<dfn>TEE</dfn>) refers to any <a>restricted operating environment</a> that 
```

Comment:
Instead of "the term X refers to Y", which makes it hard to tell whether X is an "authenticator’s matcher protection method", try to word this as "An authenticator's user matching is protected by `"external-chip"` if ..."